packages/table/src/Row.tsx

```diff
@@ -242,17 +244,13 @@ const Row = React.forwardRef(
       return renderedChildren;
     }, [children, rowHasNestedRows, isExpanded, setIsExpanded]);
 
-    // Depending on network speed, will noticeably render columns with incorrect
-    // alignment, would rather wait for proper information before rendering
-    if (!columnInfo) {
```

Comment:
Calling this out pre-emptively.

I removed this block because it was preventing rendering server-side. I tried disabling this block on the server, but it causes a mismatch between the client and server upon hydration.

Reply:
Is there another work around? For tables with large data sets this lag is very visible.

Reply:
Also interesting because this never caused a problem with the Table component rendered in our Next app

Reply:
In regards to another workaround, I tried a few things, but ultimately the solutions I tried ended up creating a mismatch in the server rendered HTML and the initial hydrated DOM. Leaving this here causes the content to not render at all on the server. Right now, the only thing that changes is alignment, which isn't perfect, but it's better than having the component not work with SSR.

Reply:
In regards to your second comment, it is interesting, though I reproduced this with Gatsby, not Next since the docs team uses Gatsby.

Reply:
Should we consider using the hook here? 

```
if (isBrowser && !columnInfo) {
}
```

Reply:
I tried exactly that, but that also had the issue of creating a mismatch between the server and client-rendered HTML.

Reply:
how come?